test/unit/query-interface/delete.test.ts

```diff
@@ -39,4 +39,116 @@ describe('QueryInterface#delete', () => {
+    expectsql(firstCall.args[0] as string, {
+      default: 'DELETE FROM `Users` WHERE `id` = \':id\'',
+      postgres: `DELETE FROM "Users" WHERE "id" = ':id' RETURNING *`,
+      mssql: `DELETE FROM [Users] WHERE [id] = N':id'; SELECT @@ROWCOUNT AS AFFECTEDROWS;`,
+      snowflake: `DELETE FROM "Users" WHERE "id" = ':id'`,
+      db2: `DELETE FROM "Users" WHERE "id" = ':id'`,
+      ibmi: `DELETE FROM "Users" WHERE "id" = ':id'`,
+    });
+  });
+
+  it('returns query with `options.returning` set to true', async () => {
+    const stub = sinon.stub(sequelize, 'queryRaw');
+    const instance = new User();
+
+    await sequelize.getQueryInterface().delete(
+      instance,
+      User.tableName,
+      { id: ':id' },
+      {
+        returning: true,
+        replacements: {
+          id: '123',
+        },
+      },
+    );
+
+    expect(stub.callCount).to.eq(1);
+    const firstCall = stub.getCall(0);
+    expectsql(firstCall.args[0] as string, {
+      default: 'DELETE FROM `Users` WHERE `id` = \':id\'',
+      postgres: `DELETE FROM "Users" WHERE "id" = ':id' RETURNING *`,
+      mssql: `DELETE FROM [Users] WHERE [id] = N':id'; SELECT @@ROWCOUNT AS AFFECTEDROWS;`,
+      snowflake: `DELETE FROM "Users" WHERE "id" = ':id'`,
+      db2: `DELETE FROM "Users" WHERE "id" = ':id'`,
+      ibmi: `DELETE FROM "Users" WHERE "id" = ':id'`,
+    });
+  });
+
+  it('returns query with fields specified in `options.returning`', async () => {
+    const stub = sinon.stub(sequelize, 'queryRaw');
+    const instance = new User();
+
+    await sequelize.getQueryInterface().delete(
+      instance,
+      User.tableName,
+      { id: ':id' },
+      {
+        returning: ['firstName'],
+        replacements: {
+          id: '123',
+        },
+      },
+    );
+
+    expect(stub.callCount).to.eq(1);
+    const firstCall = stub.getCall(0);
+    expectsql(firstCall.args[0] as string, {
+      default: 'DELETE FROM `Users` WHERE `id` = \':id\'',
+      postgres: `DELETE FROM "Users" WHERE "id" = ':id' RETURNING "firstName"`,
+      mssql: `DELETE FROM [Users] WHERE [id] = N':id'; SELECT @@ROWCOUNT AS AFFECTEDROWS;`,
+      snowflake: `DELETE FROM "Users" WHERE "id" = ':id'`,
+      db2: `DELETE FROM "Users" WHERE "id" = ':id'`,
+      ibmi: `DELETE FROM "Users" WHERE "id" = ':id'`,
+    });
+  });
+
+  it('returns query with `options.returning` set to false', async () => {
+    const stub = sinon.stub(sequelize, 'queryRaw');
+    const instance = new User();
+
+    await sequelize.getQueryInterface().delete(
+      instance,
+      User.tableName,
+      { id: ':id' },
+      {
+        returning: false,
+        replacements: {
+          id: '123',
+        },
+      },
+    );
+
+    expect(stub.callCount).to.eq(1);
+    const firstCall = stub.getCall(0);
+    expectsql(firstCall.args[0] as string, {
+      default: 'DELETE FROM `Users` WHERE `id` = \':id\'',
+      postgres: `DELETE FROM "Users" WHERE "id" = ':id'`,
```

Comment:
That counts for postgres as well here

Reply:
Differences in apostrophes couldn't be handled without special cases for dialects. In few tests(not in this PR), special cases are used for each dialects in assertions. For now, could I leave it as it is?

Reply:
You need to update default to the following so that the difference in apostrophes are used.
```ts
default: `DELETE FROM [Users] WHERE [id] = ':id'`,
```

Reply:
Got it !!